src/common/hotbar-store/__tests__/hotbar.test.ts

```diff
@@ -0,0 +1,207 @@
+/**
```

Comment:
The naming of tests here and elsewhere would benefit from a naming convention such as gherkin (ie. given, when, then). Let's talk about this elsewhere, otherwise superficial.